### DISCUSSION TITLE
如何调整，默认打开窗体的时候，所使用的终端

### DISCUSSION THREAD

**Discussion by Ivorzk** [Eugeny/tabby#6924] [Q&A]
![1659683299629](https://user-images.githubusercontent.com/7390138/183021890-82f778c4-f84b-44b7-856e-8a655c5254a6.jpg)

目前这几个终端，默认使用的是cmd，支持的命令行比较弱，我想让他默认使用  gitbash 或者 power shell 应该如何处理！

**Comment by lawweiliang** [comment]
@ioldyao
English only please.
https://github.com/Eugeny/tabby/discussions/5533

![image](https://user-images.githubusercontent.com/34936929/228172727-cc58dcc4-bdf4-4085-844a-a25cf784c145.png)

**Comment by lawweiliang** [comment]
@Ivorzk    
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
